ovs/client.go

```diff
@@ -68,12 +68,12 @@ func shellExec(cmd string, args ...string) ([]byte, error) {
 // exec executes an ExecFunc using the values from cmd and args.
 // The ExecFunc may shell out to an appropriate binary, or may be swapped
 // for testing.
-func (c *Client) exec(cmd string, args ...string) ([]byte, error) {
+func (c *Client) exec(cmd string, sudo bool, args ...string) ([]byte, error) {
```

Comment:
I don't think that disabling sudo on a per exec basis is what we want.

It seems like the caller should choose to use sudo, _or_ the TLS options.  If they specify both, that's on them, but we shouldn't have any special cases.

Reply:
The problem is the tls option is only for ovs-ofctl. ovs-vsctl still needs sudo.

Reply:
Where this is used internally in hvflowd, we only use the ofctl portions.  Realistically we should break the client in two.

I'd prefer to keep this simple and not change any of the existing sudo logic.  The ofctlFlags slice is already added to the various calls in the OpenFlow portion.